src/open_inwoner/components/templates/components/Form/FileInput.html

```diff
@@ -1,15 +1,23 @@
-{% load i18n button_tags card_tags form_tags icon_tags %}
+{% load i18n solo_tags button_tags card_tags form_tags icon_tags %}
 
-<div class="form__control file-input">
+{% get_solo 'openzaak.OpenZaakConfig' as openzaak_config %}
+
+<div class="form__control file-input" aria-live="polite">
     {% render_card direction="vertical" %}
         {% icon icon="upload" icon_position="before" outlined=True %}
-        <input class="file-input__input" id="{{ field.auto_id }}" name="file" type="file"{% if field.field.required %} required{% endif %}{% if multiple %} multiple{% endif %}>
-        <label class="button button--primary" for="{{ field.auto_id }}">
+        <input class="file-input__input" id="{{ field.auto_id }}" name="file" type="file"{% if field.field.required %} required{% endif %}{% if multiple %} multiple{% endif %} data-max-size="{{ openzaak_config.max_upload_size }}">
+        <label class="button button--primary file-input__empty" for="{{ field.auto_id }}">
```

Comment:
Can we either:

- Set the label(s) to be a an element (`file-input__label`) with a modifier set to is's meaning (`file-input__label file-input__label--empty`).
- Or make the individual names more descriptive (`file-input__label-empty`)